nexus/src/app/mod.rs

```diff
@@ -454,6 +454,16 @@ impl Nexus {
         let mid = self.samael_max_issue_delay.lock().unwrap();
         *mid
     }
+
+    // Convenience function that exists solely because writing
+    // LookupPath::new(&opctx, &nexus.datastore()) in an endpoint handler feels
+    // like too much
+    pub fn db_lookup<'a>(
+        &'a self,
+        opctx: &'a OpContext,
+    ) -> db::lookup::LookupPath {
+        db::lookup::LookupPath::new(opctx, &self.db_datastore)
+    }
```

Comment:
As I say in the comment, we can do without this, but having to refer explicitly to `nexus.datastore()` inside an endpoint handler to build the lookup feels like it violates the abstraction boundary of the `Nexus` object. No caller will ever have to make a lookup with a different datastore than the one that hangs of `nexus`.